### PR TITLE
Added Remaining v0.3.0 release files.

### PR DESCRIPTION
**Pull request by jweite-amazon**
*Issue #, if available:*

*Description of changes:*

*Testing performed:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: releases/v0.3.0/EVALUATION_DEPLOYMENT.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+```
+[Global]
+api-url = <urlOfCloudStackAPI>
+api-key = <cloudstackUserApiKey>
+secret-key = <cloudstackUserSecretKey>
+```
+After defining this in a file named cloud-config, create the environment variable with:
+
+```
+export CLOUDSTACK_B64ENCODED_SECRET=$(cat cloud-config | base64 | tr -d '\n')
```
Comment (wongni) [inline releases/v0.3.0/EVALUATION_DEPLOYMENT.md:42]: The base64 executable in some Linux distributions wraps lines by default, which causes CLOUDSTACK_B64ENCODED_SECRET to be a multi-line string. Adding a step to execute `echo $CLOUDSTACK_B64ENCODED_SECRET` may be a good idea to make sure the environment variable is a single-line string. In the case that the string is a multi-line string, adding a -w0 flag to base64 would disable the default line wrapping.

Comment (jweite-amazon) [reply releases/v0.3.0/EVALUATION_DEPLOYMENT.md:42]: Thanks @wongni, I'll create a ticket to do this for 0.3.1.

**Pull request merged**